app/server/auth/authenticate.py

```diff
@@ -10,6 +10,10 @@
 async def get_token_from_gitlab(
     code: str, client_id: str, client_secret: str, redirect_uri: str
 ) -> dict:
+    """
+    Get the token from GitLab, and authorize request to the OPA Manager
```

Comment:
```suggestion
    Get the token from GitLab and authorize the request to the OPA Manager
```

Reply:
This function retrieves only the token and doesn't authorize the OPA Manager application.